### PR TITLE
Some cleanup

### PR DESCRIPTION
**Pull request by aqw**
@mih asked me to take a look at this and edit it. A lot of my changes are easy, low controversy stuff. Some sections I took an opinionated approach and reworked them in a way that felt better to me.

The largest change (not in lines, but in approach) was an adjustment of the tone to be a bit more dry - especially in the GLM analysis section. The text strung the users along, continuing to promise them just one more step, which then always had substeps upon substeps. It's chain yanking, and the text's efforts to sympathize with the user rang false with me, since the text knows damn well what is coming, and is not "along for the ride" with the reader. Let's instead be dry, efficient, to the point, and let the reader determine whether they are exasperated or not with the number of steps.

I tried breaking up the commits a bit, so it's not one huge blob. But you know how it is with writing. :-/

Overall, it looks good, and with this cleanup, it looks pretty close to being ready (modulo whatever you have planned for it). I will open a few issues with some side comments, so that they can be discussed outside of this PR.

Comments, concerns, criticisms are welcome.

---Alex

### CODE REVIEW

File: _episodes/03-01-reproin.md [hunk 1]
```diff
@@ -397,13 +397,12 @@ First, let's convert the events.tsv file into EV3 format files.
...
-with FSL. The following command will create a working configuration from the
-template we have stored in `code/`. It uses the mighty `sed` editor. That one is
-mind-boggling, but once you know how to use it, you never want to forget about
-it again. Let's also run that through [datalad run], so we know forever that we didn't
-type this in by hand, but actually generated it from a template (that we could
-alter and then regenerate this file).
+All we have left is to configure the desired first-level GLM analysis with FSL.
+The following command will create a working configuration from the template we
+stored in `code/`. It uses the arcane, yet powerful `sed` editor. We will again
+run use [datalad run] to invoke our command, so that we store in the history
```
Comment (mih) [inline _episodes/03-01-reproin.md:403]: Note to self: Remove 'run'

**Comment by mih** [comment]
Thanks a lot -- this is a substantial improvement!

File: _episodes/03-01-reproin.md (hunk 1 above)
Comment (mih) [reply _episodes/03-01-reproin.md:403]: Done.

Comment (aqw) [reply _episodes/03-01-reproin.md:403]: Damn. My bad. Good catch. ;-)

**Comment by aqw** [comment]
@mih glad it helped. :-)

**Pull request merged**